api/openapi.yaml

```diff
@@ -237,11 +265,10 @@ components:
       properties:
         name:
           type: string
-        id:
-          type: string
-          format: uuid
         namespace:
           type: string
+        apis:
```

Comment:
@olensmar this is a reference to this comment here https://github.com/kubeshop/kuskgateway-api-server/pull/1#discussion_r824556540
I went under assumption that by service you meant apis that are "handled" by the envoyfleet instance. (I hope my wording makes some sense)